README.md

```diff
@@ -1417,6 +1418,55 @@ Server: spray-can/1.3.3
 }
 ```
 
+## GET /api/workflows/:version/query
+
+cURL:
+
+```
+$ curl "http://localhost:8000/api/workflows/v1/query?start=2015-11-01&end=2015-11-03&status=Failed&status=Succeeded"
+```
+
+HTTPie:
+
+```
+$ http "http://localhost:8000/api/workflows/v1/query?start=2015-11-01&end=2015-11-03&status=Failed&status=Succeeded"
+```
+
+Response:
+```
+HTTP/1.1 200 OK
+Content-Length: 133
+Content-Type: application/json; charset=UTF-8
+Date: Tue, 02 Jun 2015 18:06:56 GMT
+Server: spray-can/1.3.3
+
+{
+  "results": [
```

Comment:
Not sure.  I like the minimalism, but I'm not sure how nicely that would work with unmarshalling.